stories/icons/index.stories.tsx

```diff
@@ -43,33 +41,46 @@ WithControls.parameters = {
   chromatic: { disable: true },
 };
 
+/**
+ * Rough hack to determine if React function component returns null.
+ */
+const shouldRenderIcon = (Icon: valueof<typeof iconComponents>) => {
+  const renderedIcon = Icon({});
+
+  return renderedIcon !== null;
+};
+
 export const IconLibrary = () => (
   <React.Fragment>
     <Typography.Heading>Logos</Typography.Heading>
     <Style.Icons>
       {Object.entries(logoComponents).map(([name, Icon]) => (
         <Style.IconContainer key={name}>
-          <Icon width={name === WORDMARK_NAME ? 175 : 40} height={40} />
+          <Icon />
           <Style.IconLabel>{name}</Style.IconLabel>
         </Style.IconContainer>
       ))}
     </Style.Icons>
 
     <Typography.Heading>Icons</Typography.Heading>
     <Style.Icons>
-      {Object.entries(iconComponents).map(([name, Icon]) => (
-        <Style.IconContainer key={name}>
-          <Icon width={20} height={20} />
-          <Style.IconLabel>{name}</Style.IconLabel>
-        </Style.IconContainer>
-      ))}
+      {Object.entries(iconComponents).map(([name, Icon]) => {
+        if (!shouldRenderIcon(Icon)) return null;
+
+        return (
+          <Style.IconContainer key={name}>
+            <Icon />
+            <Style.IconLabel>{name}</Style.IconLabel>
+          </Style.IconContainer>
+        );
+      })}
     </Style.Icons>
 
     <Typography.Heading>Emojis</Typography.Heading>
     <Style.Icons>
       {Object.entries(emojiComponents).map(([name, Icon]) => (
         <Style.IconContainer key={name}>
-          <Icon width={48} height={48} />
```

Comment:
Since we are defining this within individual SVGs, we no longer need to explicitly pass in these values in usage (when they match the default).